tests/unit/views/pacbio/PacbioLibraries.spec.js

```diff
@@ -93,18 +92,22 @@ describe('Libraries.vue', () => {
   describe('#showAlert', () => {
     it('passes the message to function on emit event', () => {
       libraries.showAlert('show this message', 'danger')
-      expect(wrapper.find(Alert).html()).toMatch('show this message')
+      wrapper.vm.$nextTick(() => {
+        expect(wrapper.findComponent({ref: 'alert'}).html()).toMatch('show this message')
+      })
     })
   })
 
+  // TODO: Why isnt this working
```

Comment:
What isnt working ?

Reply:
It is now due to addition of sortDesc false. Proper weirdness.